test/api_test.js

```diff
@@ -69,21 +70,31 @@ describe("Api", () => {
           .eql("http://fake-server/v0/batch");
       });
 
+      it("should provide bucket endpoint", () => {
+        expect(endpoints.bucket("foo"))
+          .eql("http://fake-server/v0/buckets/foo");
+      });
+
       it("should provide collection endpoint", () => {
-        expect(endpoints.collection("toto"))
-          .eql("http://fake-server/v0/collections/toto/records");
+        expect(endpoints.collection("foo", "bar"))
+          .eql("http://fake-server/v0/buckets/foo/collections/bar");
+      });
+
+      it("should provide records endpoint", () => {
+        expect(endpoints.records("foo", "bar"))
+          .eql("http://fake-server/v0/buckets/foo/collections/bar/records");
       });
 
       it("should provide record endpoint", () => {
-        expect(endpoints.record("toto", 42))
-          .eql("http://fake-server/v0/collections/toto/records/42");
+        expect(endpoints.record("foo", "bar", 42))
+          .eql("http://fake-server/v0/buckets/foo/collections/bar/records/42");
       });
     });
 
     describe("absolute URL", () => {
       var endpoints;
 
-      beforeEach(() => endpoints = api.endpoints({fullUrl: false}))
+      beforeEach(() => endpoints = api.endpoints({ fullUrl: false }))
```

Comment:
Nit: we should keep consistent regarding inline object definitions; here we're using spaces around, later we're not anymore.